clearpath/components/component/terminal/dailymotivation.tsx

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable react/no-unescaped-entities */
+import React from "react";
+import { useEffect, useRef, useState } from "react";
+
+interface Quote {
+  quote: string;
+  author: string;
+}
+
+export default function DailyMotivation() {
+  const [quote, setQuote] = useState<Quote | null>(null);
+  const [isLoading, setIsLoading] = useState(true); // Track loading state
+  const [error, setError] = useState<string | null>(null); // Track errors
+
+  useEffect(() => {
+    const fetchQuote = async () => {
+      const apiKey = process.env.NEXT_PUBLIC_API_NINJAS_KEY;
+      if (!apiKey) {
+        setError("API key is missing. Check your .env.local file.");
+        setIsLoading(false);
+        return;
+      }
+
+      try {
+        const response = await fetch(
+          "https://api.api-ninjas.com/v1/quotes?category=happiness",
```

Comment:
Is this the only place we need this ? if yes then fine but if we will need it else then lets refactore it

Reply:
Yes this is the only place where the quotes are used; but I had spoken with @boypaida12 On how to refactor this code to fit Next.JS conventions

He needs to create a fetch function that can call the Quotes API and make it globally accessible by putting it in the utils folder